test/vault-tls.hcl

```diff
@@ -7,3 +7,6 @@ listener "tcp" {
 }
 
 disable_mlock = true
+
+default_lease_ttl = "768h"
```

Comment:
I set these to the Vault defaults, so that the test for updating these values has a documented and configured base state to assert agains